packages/ui/src/accounts/components/AccountItem/AccountItem.tsx

```diff
@@ -32,13 +32,13 @@ export const AccountItem = ({ account }: AccountItemDataProps) => {
     <AccountItemWrapper onClick={() => setDropped(!isDropped)}>
       <AccountItemWrap key={address}>
         <AccountInfo account={account} />
-        <TokenValue value={balance?.total} />
+        <TokenValue value={balance?.total} isLoading={balance?.total === undefined} />
```

Comment:
Just for consistency:
```suggestion
        <TokenValue value={balance?.total} isLoading={!isDefined(balance?.total)} />
```